source/NVDAObjects/behaviors.py

```diff
@@ -873,30 +922,6 @@ def event_alert(self):
 
 	event_show = event_alert
 
-class EditableTextWithSuggestions(NVDAObject):
-	"""Allows NvDA to announce appearance/disappearance of suggestions as text is entered.
-	This is used in various places, including Windows 10 search edit fields and others.
-	Subclasses should provide L{event_suggestionsOpened} and can optionally override L{event_suggestionsClosed}.
-	These events are fired when suggestions appear and disappear, respectively.
-	"""
-
-	def event_suggestionsOpened(self):

```

Comment:
same questions here re backwards compatibility

Reply:
These were moved to InputFieldWithSuggestions, which now EditableTextWithSuggestions inherits from.